src/Jellyfin.XmlTv/Jellyfin.XmlTv.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net8.0</TargetFrameworks>
```

Comment:
Do we also want to add net9 or only LTS?

Reply:
I've kept it on LTS as that will be supported longer and we don't update this often.